modules/installing-ocp-agent-ibm-z-kvm.adoc

```diff
@@ -37,4 +44,30 @@ $ virt-install \
    --extra-args "coreos.inst.persistent-kargs=console=tty1 console=ttyS1,115200n8" \
    --osinfo detect=on,require=off
 ----
-<1> For the `--location` parameter, specify the location of the kernel/initrd on the HTTP or HTTPS server.
\ No newline at end of file
+<1> For the `--location` parameter, specify the location of the kernel/initrd on the HTTP or HTTPS server.
+endif::pxe-boot[]
+
+ifndef::pxe-boot[]
```

Comment:
```suggestion
ifdef::pxe-boot[]
```

Reply:
@SNiemann15 I think this ifdef logic needs to stay the way it is, because right now it ensures that lines 26-47 are only displayed in the PXE boot procedure and lines 54-68 are only displayed in the regular Agent install procedure. To my understanding that's what is intended by this PR, please see the docs preview of the PR to verify that the rendered doc looks correct to you.